docs/teams/after-competition.md

```diff
@@ -3,14 +3,6 @@ original:
   authors: Fiona Gillan
   url: https://github.com/fgillan/sr-team-coord/wiki/After-competition
 ---
-## Kit Return
```

Comment:
Maybe worth keeping a comment here with a brief summary of kit return? Perhaps just that it happens at the competition event and that extended loans exist, but cross linking to the relevant other section?